efficient-r-feedback-chapter2-rcotton.md

```diff
@@ -0,0 +1,124 @@
+## General points
+
+I think it's worth making an effort between distinguishing the bits of book that are how-to tips, and the bits that are explaining underlying concepts.
+
+You also need to sell some ideas harder.  For example, describe a problem that some your advice then solves.  For example, I'm a lousy typist, and autocomplete means fewer typos.  Or, R has a lots of functions, and remembering their names can be a nightmare.  This is where `apropos` and `??` come in useful.
```

Comment:
Use a rmdnote to mention that lots of functions is 100,000's http://r4stats.com/2016/04/19/rs-growth-continues-to-accelerate/